content/en/home/timeline.md

```diff
@@ -18,9 +18,27 @@ date_format = "January 2006"
 #   Required fields are `title`, `company`, and `date_start`.
 #   Leave `date_end` empty if it's your current employer.
 #   Begin/end multi-line descriptions with 3 quotes `"""`.
+[[experience]]
+  title = "v1.0.0"
+  company = "KubeEdge v1.0.0 release"
+  company_url = "https://github.com/kubeedge/kubeedge/releases/tag/v1.0.0"
+  location = "June, 2019"
+  date_start = ""
+  date_end = ""
+  description = """
+**Features added**
+
+- Edge mesh provides service mesh capabilities on the edge. Pod-to-pod communication on the same edge node or across edge nodes in the same subnet is supported. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/edgemesh-design.md' target="_blank">&hellip; Read more</a>
```

Comment:
Point the links to the appropriate urls from the docs.kubeedge.io website.

Reply:
updated with the website links for available documentation. removed md links.